detections/container_implant_aws_detection.yml

```diff
@@ -0,0 +1,48 @@
+asset_type: AWS ECR container
+confidence: medium
+creation_date: '2020-02-20'
+data_metadata:
+  data_models:
+    - Vulnerabilities
+  data_source:
+    - AWS Cloud Trail logs 
+  data_sourcetypes:
+    - aws:cloudtrail
+  providing_technologies:
+    - AWS
+description: This searches show information on uploaded containers including source user, image id, source IP
+  user type, http user agent, region, first time, last time of operation (PutImage). These searches
+  are based on Cloud Infrastructure Data Model.
+detect:
+  splunk:
+      search: '| tstats count min(_time) as firstTime max(_time) as lastTime FROM datamodel=Cloud_Infrastructure.Compute where Compute.user_type!="AssumeRole" AND Compute.http_user_agent="AWS Internal" AND Compute.event_name="PutImage" by Compute.image_id Compute.src_user Compute.src Compute.region Compute.msg Compute.user_type | `drop_dm_object_name("Compute")`'
```

Comment:
@rosplk : we should add post filter macros in both the search and create respective macro ymls